UefiTestingPkg/FunctionalSystemTests/SmmPagingProtectionsTest/App/SmmPagingProtectionsTestApp.inf

```diff
@@ -32,21 +32,26 @@
   MdePkg/MdePkg.dec
   MdeModulePkg/MdeModulePkg.dec
   MsUnitTestPkg/MsUnitTestPkg.dec
-
+  UefiTestingPkg/UefiTestingPkg.dec
 
 [LibraryClasses]
+  BaseLib
+  BaseMemoryLib
   DebugLib
+  PcdLib
+  PrintLib
   UefiApplicationEntryPoint
   UefiBootServicesTableLib
   UnitTestLib
   UnitTestLogLib
   UnitTestPersistenceLib
   UnitTestAssertLib
-  PrintLib
 
 [Protocols]
   gEfiSmmCommunicationProtocolGuid
 
-
 [Guids]
   gEdkiiPiSmmCommunicationRegionTableGuid
+
+[FixedPcd]
+  gUefiTestingPkgTokenSpaceGuid.SkipSmmProtectionFeatures
```

Comment:
not sure i like this PCD.  I don't think SmmProtectionFeatures is very clear.  

We try to contain this info in one place.  For example maybe the SMI driver returns an unsupported when the test app requests a feature the platform doesn't support.  In the test this would then take the skipped path.  
Then any config can go into the SMI handler instead of being in both.  Just a thought